modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/ErlangProperCodegen.java

```diff
@@ -0,0 +1,543 @@
+/*
+ * Copyright 2018 OpenAPI-Generator Contributors (https://openapi-generator.tech)
+ * Copyright 2018 SmartBear Software
```

Comment:
Please remove line 3 (ref: https://opensource.stackexchange.com/questions/7300/copyright-notice-in-the-file-header-apache-v2-license/7301#7301)